lib/karaokium_web/live/admin/performance_live/new.ex

```diff
@@ -15,6 +15,7 @@ defmodule KaraokiumWeb.Admin.PerformanceLive.New do
      socket
      |> assign(:performance, %Performance{})
      |> assign(:page_title, "New Performance")
-     |> assign(:karaoke_id, karaoke_id)}
+     |> assign(:karaoke_id, karaoke_id)
+     |> assign(:permissions, socket.assigns.current_user.permissions)}
```

Comment:
Where are you using this?

Reply:
I am using this on the same folder, on the file new.html.heex. I made this in order to have the same page returning to 2 different pages. This is important when we have normal users, that doesn't have any permission, and they want to register their team, their return page will be the normal karaoke page. The rest users that have permissions will go directly to the admin page when they finished registering some team. I made this for the reason to not creating another new.ex and new.html.heex which only difference would be the return page.